app/views/lists/index.html.erb

```diff
@@ -1,2 +1,26 @@
-<h1>Lists#index</h1>
-<p>Find me in app/views/lists/index.html.erb</p>
+<p><%= link_to('Add a new list', new_list_path, :class => 'action new') %></p>
+
+<% flash.each do |type, msg| %>
+  <script type="text/javascript" charset="utf-8">
+      alert("<%= msg %>");
+  </script>
+<% end %>
+
+<h1>Lists:</h1>
+
+<table>
+  <tr>
+    <th>Title</th>
+    <th>Action</th>
+  </tr>
+
+  <% @lists.each do |list| %>
+    <tr>
+      <th><%= list.title %></th>
+      <th><%= link_to('Show', list_path(list)) %></th>
+      <th><%= link_to('Edit', edit_list_path(list)) %></th>
+      <th><%= link_to('Delete', delete_list_path(list)) %></th>
```

Comment:
instead of the added `delete` route and action,
can we open an alert/pop-up when someone clicks on the delete link here?

Reply:
Yes, of course. I will change it.